packages/new-polymath-issuer/src/pages/Home.tsx

```diff
@@ -61,14 +72,72 @@ export class ContainerBase extends Component<Props> {
     );
   };
 
+  public startCreateCheckpoint = () => {
+    this.props.dispatch(
+      createCheckpointStart({
+        securityTokenSymbol: 'A0T0',
+      })
+    );
+  };
+
+  public startCreateDividendDistribution = () => {
+    this.props.dispatch(
+      createErc20DividendDistributionStart({
+        securityTokenSymbol: 'A0T0',
+        maturityDate: new Date(),
+        expiryDate: new Date('10/10/2025'),
+        erc20Address: '0xf12b5dd4ead5f743c6baa640b0216200e89b60da',
+        name: 'My Dividend Distribution',
+        amount: new BigNumber('10000'),
+        checkpointId: 1,
+        excludedAddresses: [],
+        pushPaymentsWhenComplete: true,
+      })
+    );
+  };
+
+  public checkForValidity = async () => {
+    console.log(
+      '0x8CdaF0CD259887258Bc13a92C0a6dA92698644C0 VALID: ',
+      await polyClient.isValidErc20({
+        address: '0x8CdaF0CD259887258Bc13a92C0a6dA92698644C0',
+      })
+    );
+
+    console.log(
+      '0xf17f52151EbEF6C7334FAD080c5704D77216b732 VALID: ',
+      await polyClient.isValidErc20({
+        address: '0xf17f52151EbEF6C7334FAD080c5704D77216b732',
+      })
+    );
+
+    console.log(
```

Comment:
I guess having this here is fine since it is just demo code still. We'll probably want to get rid of this Home file soon though.

Reply:
Yeah, for now it's useful to test procedures on the fly